hack/upgrade-test.sh

```diff
@@ -223,9 +221,12 @@ ${CMD} wait pod $CATALOG_OPERATOR_POD --for condition=Ready -n openshift-operato
 Msg "verify the subscription's currentCSV and installedCSV have moved to the new version"
 
 sleep 60
-HCO_OPERATOR_POD=`${CMD} get pods -n kubevirt-hyperconverged | grep hco-operator | head -1 | awk '{ print $1 }'`
 ${CMD} get pods -n kubevirt-hyperconverged
-${CMD} wait pod $HCO_OPERATOR_POD --for condition=Ready -n kubevirt-hyperconverged --timeout="1200s"
+${CMD} wait deployment ${HCO_DEPLOYMENT_NAME} --for condition=Available -n ${HCO_NAMESPACE} --timeout="1200s"
+
+Msg "verify new operator version"
+HCO_VERSION=$( ${CMD} get hco ${HCO_RESOURCE_NAME} -n ${HCO_NAMESPACE} -o jsonpath="{ .status.versions[?(@.name=='operator')].version }")
+[[ "${LATEST_VERSION}" == "${HCO_VERSION}" ]]
```

Comment:
This statement does not have any effect on the script flow.
The script will continue either it's true of false.
Maybe put it in an `if [...]` block